crates/sui-core/src/authority/execution_time_estimator.rs

```diff
@@ -348,10 +348,12 @@ impl ExecutionTimeObserver {
         let transaction = ConsensusTransaction::new_execution_time_observation(
             ExecutionTimeObservation::new(epoch_store.name, to_share),
         );
-        if let Err(e) = self
-            .consensus_adapter
-            .submit_to_consensus(&[transaction], &epoch_store)
-        {
+
+        if let Err(e) = self.consensus_adapter.submit_best_effort(
+            &transaction,
+            &epoch_store,
+            Duration::from_secs(5),
+        ) {
             if !matches!(e, SuiError::EpochEnded(_)) {
                 epoch_store
                     .metrics
```

Comment:
this metric is now sorta confusing/misleading, because the cases where consensus is overloaded will be counted here, but the cases where timeout causes the message to be dropped wlil not be counted here and will instead show up in `sequencing_best_effort_timeout`

Reply:
i actually don't think this metric could ever be incremented before - by my reading, EpochEnded is the only error that can actually be returned from submit_to_consensus

as for the timeout case - the timeouts were already effectively possible in that the same code that submit_best_effort relies on could also end up retrying forever in when called via submit_to_consensus. The only difference is that with the timeout we can actually notice it in a metric.